### PR TITLE
Improve footer link contrast

### PR DESCRIPTION
**Pull request by thenaterhood**
The blue on gray is a little hard to see in the footer in some circumstances, like if you use f.lux or redshift. This lightens it up a hair to make it a little bit better (but not so much as to totally change the look).

# Before:
image

# After:
image

### CODE REVIEW

**Comment by jwflory** [comment]
+1, looks great! :heart_eyes:

**Comment by ct-martin** [comment]
I'm finding the new color really hard to read normally (contrast issue?). I have no issue with changing the color, something else might be better though. Using off-white, maybe with an underline, would probably be more readable.

**Comment by thenaterhood** [comment]
Sure, works for me! I can try out some other colors and see how they do.

**Comment by Tjzabel** [comment]
@ct-martin the lighter color is harder to read for you than the old color?

For me personally, I have no qualms with the change in color. I can see that it may not be perfect, but I'm not sure the best color options here. If we do a shade of grey/white, I would want the links underlined or italicized in some way to distinguish them from the normal text.

Not sure what the best solution is :man_shrugging:

**Comment by thenaterhood** [comment]
It's all simple CSS, so easy enough either way :)

We could even try out a lighter shade of blue if we like the blue.

**Comment by ct-martin** [comment]
(cc @Tjzabel )

So TL;DR color theory, there needs to be a visible distinction with the "decorated" text and both the normal text as well as the background. The most robust distinction purely on the basis of color is contrast (grayscale difference between color values). This works with the white and medium-dark gray, however, it's somewhat more awkward between the white and gray (in part b/c ratios not absolute values).

The issue I had with Nate's was that I found I was glazing over the text as if it didn't exist, which gets to why I didn't have an issue with the current styling. The current styling has a very distinctly different saturation. Grayscale values are zero or near-zero saturation, whereas the blue being used had a very strong saturation yet different enough from black that it could be distinguished as a different contrast. There is an accessibility issue here as Nate discovered, which is what color-blindness & color-modifying filters such as red-shift reduce this saturation. However, Nate's color has a very low saturation while maybe also not having a good contrast with the background gray. When using a red-shift it changed the color enough to add some more contrast based on my observation, but without the red-shift my issue happens. I didn't check, but there are also calculators online for contrast ratios and accessibility.

Anyway, here are the thoughts I had when talking to Nate in-person:
* Something maybe closer to an off-white than, and more saturated than, the light blue (would need to play with; I'd ballpark around `hsl(240, 100, [85 or 90])` if I had to make a guess without checking)
  * To break that color down, Hue 240 is blue, Saturation 100 would create the first visual distinction, and Lightness 85 or 90 should be low enough that it's distinct from the white but high enough to have contrast with the gray
    * Going higher than 90 for lightness will be too close to the white, and I'm not sure how low we can go not I would guess not below 80 at lowest
    * As I said, I didn't check this so take it with a grain of salt
* White text & using a link icon (we have Font Awesome on the site already)
  * The icon could be before the link as the bullet itself, or after the link like Wikipedia's external links
* White text with underline and/or italics (note that italics have another semantic meaning though and we should probably avoid them)

That was probably a lot longer of an answer than was desired... :man_shrugging: 

TL;DR - the current color has an accessibility issue, and Nate's does as well. My POV is that we should not change one non-accessibly-friendly color for another. If about what I ballparked for a color doesn't seem to work with and without red-shift I think we should move on to white with something else (preferably an icon or underline) stylistically denoting it as a link. None of them would be hard to do; it's either simple CSS or using a touch of Bootstrap.

**Comment by jwflory** [comment]
@ct-martin Could you open a new pull request with better color choices?

**Comment by ct-martin** [comment]
@jwflory maybe; it'd depend on if I have time early next week (since this weekend is homecoming and I'll probably have family visiting). I can mock something simple but I don't have the bandwidth to pursue testing implementations right now beyond giving feedback.

Here's a two-second mock of `hsl(240,100%,85%)` (aka `#B3B3FF`):
image

If that seems to work for people it could be used; if not then we should probably look at other decoration. Personally, I'm not completely in love with that blue and I'm thinking it may make more sense to prepend a link icon and an underline on hover.

**Comment by ct-martin** [comment]
Aside: for those interested:

Current color:
image

Nate's color:
image

My color:
image

**Comment by Tjzabel** [comment]
@nlMeminger I didn't know we were in the UK :smile:

**Comment by nlMeminger** [comment]
Don’t make fun of me for spelling words correctly. Using the correct
spellings in my favourite thing besides going to the theatre

**Comment by jrtechs** [comment]
@thenaterhood @ct-martin what do we want to do with this PR? I'm fine with merging this branch after we update it with the purple color @ct-martin mentioned.
```
hsl(240,100%,85%) (aka #B3B3FF):
```

**Comment by ct-martin** [comment]
@jrtechs I still think a white with styling would be better. I'll be at FOSS Hours this week, could you remind me to play with it then?

**Comment by thenaterhood** [comment]
Play with it! I'm happy to change it to whatever works better :)

**Comment by thenaterhood** [comment]
Here's white with an underline (hover removes the underline). See what you think!
image

**Comment by Tjzabel** [comment]
Here's a slightly more detailed screenshot of if we change the footer to all white:

image


This isn't an awful option, but seems to make the whole bottom footer super bland.

**Comment by thenaterhood** [comment]
That's the downside of making things white! The purply color seems fine too, visibility-wise.

**Comment by Tjzabel** [comment]
What does everyone think of combining the two ideas of underlining with lighter blue?

image

This was done via:
```
footer a {
    color: #B3B3FF;
    text-decoration: underline;
}
footer a:hover {
    color: #B3B3FF;
    text-decoration: none;
}
```

**Comment by ct-martin** [comment]
Doing both LGTM. Sorry I didn't get a chance to play with this and thanks for picking it up. Please consider making a PR on the FOSS site as well since it has the same issue and linking to this thread

**Comment by Tjzabel** [comment]
Cool. @thenaterhood I added a commit to change the color to the lighter purple while keeping the underlining. Merging :tada:
